distribution/helm/vector/templates/secret.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Secret
```

Comment:
Is there any reason that configuration is kept as a secret?
The approach that worked fine is to keep configuration as a configmap and in case some sensitive data is needed in the config, extrapolation can be used and secrets set as env variables.

Reply:
It was done this way in the filebeat chart, but I agree that the secrets would ideally be separate from the config.